_data/gremien.yml

```diff
@@ -2,38 +2,40 @@ vorstand:
   - name: "Geschäftsführender Vorstand"
     email: "vorstand@pep-dortmund.org"
     people:
-      - name: "Kevin Schmitz"
+      - name: "Christian Arauner"
         role: "1. Vorsitzender"
-        image: "/images/vorstand/kevinS.jpg"
-      - name: "Karl Schiller"
+        image: "/images/vorstand/christian_Arauner.jpg"
+      - name: "Rene-Marcel Lehner"
         role: "2. Vorsitzender"
-        image: "/images/vorstand/karl.jpg"
-      - name: "Lars Bollmann"
+        image: "/images/vorstand/rene.jpg"
+      - name: "Anno Knierim"
         role: "Finanzreferent"
-        image: "/images/vorstand/lars.jpg"
+        image: "/images/vorstand/anno.jpg"
 
   - name: "Absolventenfeier"
     email: "koordination@pep-dortmund.org"
     people:
-      - name: "Antonia Joëlle Bock"
-        image: "/images/vorstand/antonia.jpg"
+      - name: "Mira Arndt"
+        image: "/images/vorstand/mira.jpg"
 
   - name: "Ressourcenverwaltung"
     email: "ressourcenverwaltung@pep-dortmund.org"
     people:
-      - name: "Maximilian Linhoff"
-        image: "/images/vorstand/max.jpg"
+      - name: "Lars Bollmann"
+        image: "/images/vorstand/lars.jpg"
 
   - name: "Begabtenförderung"
     email: "begabtenfoerderung@pep-dortmund.org"
     people:
       - name: "Thorben Menne"
         image: "/images/vorstand/thorben.jpg"
+      - name: "Maximilian Linhoff"
+        image: "/images/vorstand/max.jpg"
 
   - name: "Fortbildungen"
```

Comment:
didn't we change this to "Workshops" last year in the MV?

Reply:
good point